tests/cephfs/snapshot_clone/cg_snap_test.py

```diff
@@ -0,0 +1,550 @@
+                        validate=True,
+                        group_name=group_name,
+                    )
+            else:
+                for snap_name in snap_list:
+                    fs_util.remove_snapshot(
+                        client, fs_name, qs_member, snap_name, validate=True
+                    )
+
+        if cg_test_io_status.value == 1:
+            log.error(
+                f"CG IO test exits with failure during quiesce lifecycle with await on qs_set-{qs_id_val}"
+            )
+            test_fail = 1
+
+    log.info("Test quiesce without --await option")
+    for qs_set in qs_sets:
+        client_mnt_dict = {}
+        qs_member_dict1 = cg_snap_util.mount_qs_members(client1, qs_set, fs_name)
+        client_mnt_dict.update({client1.node.hostname: qs_member_dict1})
+        qs_member_dict2 = cg_snap_util.mount_qs_members(client2, qs_set, fs_name)
+        client_mnt_dict.update({client2.node.hostname: qs_member_dict2})
+
+        log.info(f"Start the IO on quiesce set members - {qs_set}")
+
+        cg_test_io_status = Value("i", 0)
+        io_run_time = 100
+        p = Thread(
+            target=cg_snap_io.start_cg_io,
+            args=(clients, qs_set, client_mnt_dict, cg_test_io_status, io_run_time),
+        )
+        p.start()
+        time.sleep(45)
+        repeat_cnt = 5
+        snap_qs_dict = {}
+        for qs_member in qs_set:
+            snap_qs_dict.update({qs_member: []})
+        i = 0
+        while i < repeat_cnt:
+            if p.is_alive():
+                log.info(f"Quiesce Lifecycle : Iteration {i}")
+                # time taken for 1 lifecycle : ~5secs
+                rand_str = "".join(
+                    random.choice(string.ascii_lowercase + string.digits)
+                    for _ in list(range(3))
+                )
+                qs_id_val = f"cg_test1_{rand_str}"
+                log.info(f"Quiesce the set {qs_set}")
+                qs_output = cg_snap_util.cg_quiesce(
+                    client,
+                    qs_set,
+                    qs_id=qs_id_val,
+                    if_await=False,
+                    timeout=300,
+                    expiration=300,
+                )
+                log.info("Verify quiesce cmd response has status as quiescing")
+
+                if qs_output["sets"][qs_id_val]["state"]["name"] == "QUIESCING":
+                    log.info("Verified quiesce state is QUIESCING without await option")
+                else:
+                    raise Exception(
+                        f"quiesce state of set_id {qs_id_val} is not QUIESCING"
+                    )
+                log.info("Wait for quiesced status")
+                end_time = datetime.datetime.now() + datetime.timedelta(seconds=300)
+                qs_state_verified = 0
+                while (datetime.datetime.now() < end_time) and (qs_state_verified == 0):
+                    qs_query = cg_snap_util.get_qs_query(client, qs_id=qs_id_val)
```

Comment:
can the output of "qa_query" be logged here as debug. To triage if the quiesced state is not attained within the set timeout.

Reply:
noted